pulp_rpm/app/modelresource.py

```diff
@@ -471,11 +471,15 @@ def set_up_queryset(self):
             UpdateCollectionPackages specific to a specified repo-version.
 
         """
-        return UpdateCollectionPackage.objects.filter(
-            update_collection__in=UpdateCollection.objects.filter(
-                update_record__in=UpdateRecord.objects.filter(pk__in=self.repo_version.content)
+        return (
+            UpdateCollectionPackage.objects.filter(
+                update_collection__in=UpdateCollection.objects.filter(
+                    update_record__in=UpdateRecord.objects.filter(pk__in=self.repo_version.content)
+                )
             )
-        ).order_by("pulp_id")
+            .distinct("name", "epoch", "version", "release", "arch", "sum")
```

Comment:
I'm pretty sure in this case we want it to be unique by NEVRA, yes?

Reply:
I believe so. But sum seems like best candidate to be in ``order_by()`` - and that must be one of the names in ``distinct``.